less/cards.less

```diff
@@ -429,6 +429,26 @@
   color: transparent;
 }
 
+.enrage {
+  background-image: url("/img/enrage.png");
+  line-height: 21px;
+}
+
+.enrage.single {
```

Comment:
This is duplicated so should go in a single “.single” class. But, more than that, this whole hiding should be done in the jsx not the css

Reply:
what do you mean in JSX?
JSX should hold the logic of not showing the number?

Reply:
Fixed